wiki/en/en-Hardware-Setup.md

```diff
@@ -56,24 +56,24 @@ You're done! Have fun!
 Refer to the Linux [client install guide](Installation-for-Linux#configure-jack-with-qjackctl).
 
 ## Linux: Low Latency Kernels for Jamulus
-You might want to install [Ubuntu Studio](https://ubuntustudio.org/) it adds a second options in your boot menu for a low-latency kernel. The key of successful jamming is "low latency" between servers and connected Jamulus clients. If the underlying Linux system is started with a low-latency then it has a positive impact on latency for your Jamulus Sessions.
+You might want to install [Ubuntu Studio](https://ubuntustudio.org/). It adds a second option in your boot menu for a low-latency kernel. The key to successful jamming is "low latency" between servers and connected Jamulus clients. If the underlying Linux system is started with a low-latency kernel then it has a positive impact on latency for your Jamulus sessions.
 
 
 ## Points to note about inputs
 
-- If you want to connect 2 or more sources (eg voice + synth + guitar), it is important to note that Jamulus currently handles only input 2 channels (L/R). So the hardware being used must provide a mixed-down stereo output to Jamulus.
+- If you want to connect 2 or more sources (e.g. voice + synth + guitar), it is important to note that Jamulus currently handles only 2 input channels (L/R). So the hardware being used must provide a mixed-down stereo output to Jamulus.
```

Comment:
"Alternatively run two instances of Jamulus, if your hardware input supports it.  Or use virtual audio inputs to Jamulus for each hardware input."

I don't mind keeping it simple - but the original is wrong in the sense it states that no other solution is possible.

Reply:
I was mainly sticking to revising existing text - but if the rest of you think that extra info should go in, I'll add it.

Reply:
I know what you mean - and I'm not going to hold up this improvement just because it doesn't get us 100% perfection in one hit :).

Reply:
Done.

Reply:
Done